up2-leds/sample.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "UP Squared LEDs",
```

Comment:
I think the name should differentiate it from Onboard Blink.

Reply:
Would "UP Squared built-in LEDs" be more clear?

Reply:
Yes